support/index.js

```diff
@@ -25,7 +25,16 @@ Cypress.on('window:before:load', win => {
 });
 
 before(() => {
-  if (Cypress.env('EXTENSION') === 'metamask' && !Cypress.env('SKIP_EXTENSION_SETUP')) {
-    cy.setupMetamask();
+  if (!Cypress.env('SKIP_EXTENSION_SETUP')) {
+    switch (Cypress.env('EXTENSION')) {
+      case 'metamask':
+        cy.setupMetamask();
+        break;
+      case 'keplr':
+        cy.setupWallet();
+        break;
+      default:
+        throw new Error(`Unknown extension: ${Cypress.env('EXTENSION')}`);
+    }
   }
```

Comment:
I suggest not throwing an error. In the Metamask setup, if the required environment variables are missing, the tests continue to run. The setup relies entirely on the `SKIP_EXTENSION_SETUP` variable. I feel we should maintain the previous behavior of not throwing an error. 

If `SKIP_EXTENSION_SETUP` is falsy, regardless of whether Metamask or Keplr is used, the test cases should run.

Reply:
this is in case no `EXTENSION` is provided. in that case we are already throwing an error, so I just added one here as well

Reply:
If we throw an error here, the test cases won't run. Yeah, we're already validating `EXTENSION` at other places but over here if `SKIP_EXTENSION_SETUP` is falsy, we should allow the test cases to execute regardless.

Reply:
if the SKIP_EXTENSION_SETUP var is falsy. it will not throw an error.
It will only throw an error if the EXTENSION env var is not provided.

which makes sense considering if the user is not skipping the setup, they want to setup *some* extension.

Reply:
But doesn't it make it vague? We're already handling `EXTENSION` value at other places. The initial flow was that if `SKIP_EXTENSION_SETUP` is true, we set up the wallet else not. We added `EXTENSION` to distinguish between Metamask and Keplr. Now, we're adding a block to throw an error, which won't execute at all, because if the `EXTENSION` is not valid, it would've been handled way before reaching this snippet.

Reply:
removed the default case